src/Command/CreateControllerCommand.php

```diff
@@ -52,12 +54,15 @@ public function execute(InputInterface $input, OutputInterface $output) {
      * @param $name
      * @throws \GuzzleHttp\Exception\GuzzleException
      */
-    private function buildController($name) {
-        $response = $this->client->
-            request('GET', 'https://filebin.net/8370wdbbewlgfi94/templateController.php?t=mzggn2rt')
-            ->getBody();
+    private function buildController($name, $className) {
```

Comment:
Please use type hints for every method you create, it helps to write more stable code. 
Next question: why it is called "build"? I would say, this method *generates* it, why not to call it like this?
And again, `$name` doesn't fully reflect the purpose of the variable. 
So it should look something like this:
```suggestion
    private function generateController(string $fileName, string $className) {
```